platform-coop-toolkit/coop-map-directory-index
# feat: allow individual profile to be edited after creation

## Description

Allows one's individual profile to be edited after it's created.

- [x] Basic Information

## Steps to test

1. Check out branch.
2. Visit your profile
3. Test both the "Edit Basic Information" and "Edit Overview" links.

**Expected behavior:** Editing all data works as expected.

## Additional information

Not applicable.

## Related issues

Not applicable.
